repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -747,9 +749,11 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
 
 object KeystoneV2Filter {
 
-  private final val SYSTEM_MODEL_CONFIG = "system-model.cfg.xml"
-  private final val DEFAULT_CONFIG = "keystone-v2.cfg.xml"
-  private final val X_AUTH_PROXY = "Proxy"
+  private final val SystemModelConfig = "system-model.cfg.xml"
+  private final val DefaultConfig = "keystone-v2.cfg.xml"
+  private final val XAuthProxy = "Proxy"
```

Comment:
Yeah, i figured i'd clean it up since we've started trying to be consistent with going with the scala style in scala code. It is one of those things that's different for no reason that i can see.